### ISSUE TITLE
LIVENODING 926 / Alpha Over Chain Reaction

### ISSUE THREAD

**Issue by enzyme69** [enzyme69/blendersushi#299]
<img width="299" alt="screen shot 2018-01-26 at 10 09 46 pm" src="https://user-images.githubusercontent.com/3952687/35437253-b6ab4d4c-02e5-11e8-9f9a-ce3ef0523a31.png">
<img width="903" alt="screen shot 2018-01-26 at 10 09 15 pm" src="https://user-images.githubusercontent.com/3952687/35437254-b6dce172-02e5-11e8-8a3f-b2e3d56126bd.png">
<img width="888" alt="screen shot 2018-01-26 at 10 08 48 pm" src="https://user-images.githubusercontent.com/3952687/35437256-b70d64c8-02e5-11e8-9f86-9d4f56afc1e6.png">

[overlap_render_004_2018_01_26_11_10.zip](https://github.com/enzyme69/blendersushi/files/1667514/overlap_render_004_2018_01_26_11_10.zip)

1.
Basically you can start with a sequence of render, let say 100 frames of object moving in 3D space.

2. Based on input from 1, you make 2 render A, B, at the same time one get offset and alpha over the other.

3. The result is combined into render C.
